### DISCUSSION TITLE
I followed all the checklists in the Wiki but getting an error.

### DISCUSSION THREAD

**Discussion by tharris9d** [LizardByte/RetroArcher-plex#106] [Support] [answered]
Windows 11, Plex Media Server
From the retroarcher.log file.
`**INFO - cannot use existing database.json... maybe it does not exist yet... maybe it is corrupted...`**
I am using ffmpeg, Open Stream is running, Nintendo 64 folder name with roms in it.
[retroarcher.log](https://github.com/RetroArcher/RetroArcher.bundle/files/7585113/retroarcher.log)

Any tips and tricks would be appreciated

**Comment by ReenigneArcher** [comment, marked as answer]
First time you run the scanner you will see this message. It's not necessarily an error.

RetroArcher uses ffmpeg that is included with the install.

If you'd like me to review the logs, please post them as a gist and include the link to them. https://github.com/RetroArcher/RetroArcher.bundle/wiki/Log-Files

**Reply by tharris9d** [reply]
Thanks for the support on Discord. The videos started generating once I changed to the root rom folder.
